pom.xml

```diff
@@ -0,0 +1,205 @@
+		<plugin.git_commit_id.version>5.0.0</plugin.git_commit_id.version>
+	</properties>
+
+	<dependencies>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+			<exclusions>
+				<exclusion>
+					<groupId>org.springframework.boot</groupId>
+					<artifactId>spring-boot-starter-logging</artifactId>
+				</exclusion>
+			</exclusions>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-log4j2</artifactId>
+		</dependency>
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-actuator</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.cloud</groupId>
+			<artifactId>spring-cloud-starter-gateway</artifactId>
+		</dependency>
+
+		<dependency>
+			<groupId>org.projectlombok</groupId>
+			<artifactId>lombok</artifactId>
+			<version>${lombok.version}</version>
+		</dependency>
+
+		<dependency>
+			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-test</artifactId>
+			<scope>test</scope>
+		</dependency>
+	</dependencies>
+	<dependencyManagement>
+		<dependencies>
+			<dependency>
+				<groupId>org.springframework.cloud</groupId>
+				<artifactId>spring-cloud-dependencies</artifactId>
+				<version>${spring-cloud.version}</version>
+				<type>pom</type>
+				<scope>import</scope>
+			</dependency>
+		</dependencies>
+	</dependencyManagement>
+
+	<build>
+		<finalName>app</finalName>
+		<plugins>
+			<plugin>
+				<groupId>org.springframework.boot</groupId>
+				<artifactId>spring-boot-maven-plugin</artifactId>
+				<executions>
+					<execution>
+						<goals>
+							<goal>repackage</goal>
+						</goals>
+						<configuration>
+							<classifier>spring-boot</classifier>
+							<mainClass>org.fairdatatrain.fairdatastation.Application</mainClass>
+						</configuration>
+					</execution>
+					<execution>
+						<id>build-info</id>
+						<goals>
+							<goal>build-info</goal>
+						</goals>
+					</execution>
+				</executions>
+			</plugin>
+			<plugin>
+				<groupId>com.mycila</groupId>
+				<artifactId>license-maven-plugin</artifactId>
+				<version>${plugin.license.version}</version>
+				<configuration>
+					<header>com/mycila/maven/plugin/license/templates/MIT.txt</header>
+					<properties>
+						<owner>FAIR Data Team</owner>
+					</properties>
+					<mapping>
+						<java>JAVADOC_STYLE</java>
+					</mapping>
+					<excludes>
+						<exclude>pom.xml</exclude>
+						<exclude>**/maven.config</exclude>
+						<exclude>**/*.ttl</exclude>
+						<exclude>**/*.yml</exclude>
+						<exclude>**/*.sparql</exclude>
+						<exclude>**/*.properties</exclude>
+						<exclude>**/*.xml</exclude>
+						<exclude>**/*.json</exclude>
+						<exclude>LICENSE</exclude>
+						<exclude>.gitignore</exclude>
+						<exclude>.dockerignore</exclude>
+					</excludes>
+				</configuration>
```

Comment:
Maybe we can upgrade to the `4.1` way of using the plugin (using `licenseSets`), and move to include vs exclude (less to maintain), something like

```java
<includes>
  <include>pom.xml</include>
  <include>**/*.java</include>
</includes>
```
(any other files that make sense to put an header on?)

Reply:
Updated and included also `Dockerfile`... that is what we currently use and I did not found any *best practice* or recommendation to include also something else 🤷🏻‍♂️